lru.d.ts

```diff
@@ -56,3 +56,8 @@ export class LRUCache<K,V> {
   // Returns a human-readable text representation
   toString() : string;
 }
+
+declare module "lru-fast" {
```

Comment:
"-fast" is only a npm disambiguation suffix. The module is imported as "lru". Maybe this is an effect from using it with NPM (which is not recommended since this is barely a library, but more a small piece of code that should be copied into the target source tree.)